### DISCUSSION TITLE
Can we expect a new release any time soon? ☺️

### DISCUSSION THREAD

**Discussion by Oliver-Piorun** [angularsen/UnitsNet#1408] [General]
![grafik](https://github.com/angularsen/UnitsNet/assets/15194243/818d770c-0392-472e-8413-b61f129c7c6c)

Hey! We are really looking forward to a new release.
Do you think it would be feasible?

Thanks in advance

**Comment by angularsen** [comment]
Hi, any feature or bugfix in particular you are waiting for?

Pre-release nugets are already available for v6 if that is what you are waiting for.
https://www.nuget.org/packages/UnitsNet/6.0.0-pre007

We still want to decide on a few things before we make v6 stable, mainly whether to do #1377 or not.

**Reply by angularsen** [reply]
I realize a few changes were still not available as nugets, I just pushed 5.53 and 6.0.0-pre008.
Let me know if this helps.

[Release UnitsNet/5.53.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.53.0)

[Release UnitsNet/6.0.0-pre008 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F6.0.0-pre008)

**Reply by angularsen** [reply]
And also merged in a few things from master to v6, and shipped as 6.0.0-pre009

[Release UnitsNet/6.0.0-pre009 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F6.0.0-pre009)

**Reply by Oliver-Piorun** [reply]
Thanks a lot for your quick reply and those helpful releases! We were mainly looking for #1392.

Have a good one 😊

**Reply by angularsen** [reply]
Great, thanks!
